src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/search.html.twig

```diff
@@ -15,6 +15,9 @@
             {% endfor %}
         </select>
         <div class="clear-fix"></div>
+        <label for="status_code">Status</label>
```

Comment:
This is the status code but In the search result table, there is just "Status" label